independent-publisher-connectors/Postman/apiDefinition.swagger.json

```diff
@@ -0,0 +1,920 @@
+{
+  "swagger": "2.0",
+  "info": {
+    "version": "1.0.0",
+    "title": "Postman",
+    "description": "Postman is a platform for building and using APIs. Postman simplifies each step of the API lifecycle and streamlines collaboration.",
```

Comment:
@fordosa90 is it possible to remove the word "API" from the description? Please see [here ](https://learn.microsoft.com/en-us/connectors/custom-connectors/certification-submission#write-a-description-for-your-connector)and [here](https://learn.microsoft.com/en-us/connectors/custom-connectors/connector-string-guidance#connector-info).

Reply:
@Amjed-Ayoub I would say no, as that is their official description and that is actually indeed what Postman does. I can open up the API to "application programming interface" but that is the definition what Postman does. "API lifecycle" for example is meaningless for target community when written "application programming interface lifecycle"